rest/config.go

```diff
@@ -123,7 +123,7 @@ type DbConfig struct {
 	RevsLimit                    *uint32                        `json:"revs_limit,omitempty"`                   // Max depth a document's revision tree can grow to
 	ImportDocs                   interface{}                    `json:"import_docs,omitempty"`                  // false, true, or "continuous"
 	ImportFilter                 *string                        `json:"import_filter,omitempty"`                // Filter function (import)
-	Shadow                       *ShadowConfig                  `json:"shadow,omitempty"`                       // External bucket to shadow
+	Shadow                       *ShadowConfig                  `json:"shadow,omitempty"`                       // This is where the ShadowConfig used to be.  If found, it should throw an error
```

Comment:
This is reasonable given the circumstances, but it reminds me that we need to sort out the performance issues so that we can upgrade to go 1.10 to get the DisallowUnknownFields support in encoding/json.

Reply:
Good to know!  I was thinking we'd need to use a json schema for that, but that seems lighter weight.